### DISCUSSION TITLE
Delete samples

### DISCUSSION THREAD

**Discussion by zranz** [nzbgetcom/nzbget#415] [General]
Hi I would like samples to be deleted automatically. There are [plugin scripts](https://github.com/clinton-hall/GetScripts) out there, are they included in the docker images?

What would be the recommended way not to clutter downloads with samples? Thanks!

**Comment by dnzbk** [comment]
The `SETTINGS`->`ExtCleanupDisk` and `SETTINGS`->`UnpackIgnoreExt` options should do what you're looking for.

**Comment by dnzbk** [comment]
Oh, wait, that's not quite what you're looking for, I apologize.
As you mentioned, there are third-party plugins that can achieve this. They’re not included in the Docker image by default, you can manually install them by using [docker exec](https://docs.docker.com/reference/cli/docker/container/exec/) to access the container and placing the extensions in your `${ScriptDir}`.
Alternatively, you can wait until I add support for these extensions to the extension manager for easy installation.

**Comment by Anunnaki-Astronaut** [comment]
Hi! I actually just created a modern NZBGet extension that does exactly this. 

**RemoveSamples** - automatically detects and removes sample files and directories before Sonarr/Radarr/Lidarr/Prowlarr see them.

Features:
- Modern dropdown UI (no typing "yes/no")
- Removes both sample files AND directories  
- Configurable size thresholds for video/audio
- Pattern-based detection + size-based detection

Repository: https://github.com/Anunnaki-Astronaut/RemoveSamples-NZBGet

Just submitted it to NZBGet for inclusion in the official extension manager. Should solve the exact problem you were asking about!
